app/ide-desktop/lib/file-watcher/bundle.ts

```diff
@@ -0,0 +1,42 @@
+/** @file Entry point for the bundler. */
```

Comment:
why we need `bundle.ts`? Can't we just use cmd for that? Is custom bundler needed here?

Reply:
not needed, still from my "experimentation phase", removed it.